runtime/runtime/src/actions.rs

```diff
@@ -392,13 +394,28 @@ pub(crate) fn action_implicit_account_creation_transfer(
     actor_id: &mut AccountId,
     account_id: &AccountId,
     transfer: &TransferAction,
+    #[cfg(feature = "protocol_feature_access_key_nonce_for_implicit_accounts")]
+    current_block_height: BlockHeight,
```

Comment:
Might want to make this function unconditionally take `current_block_height` as an argument to make it easier to read and use